### ISSUE TITLE
Wrong texture in Hirule Field

### ISSUE THREAD

**Issue by ninobaldan** [GhostlyDark/OoT-Reloaded#2]
Hi! First of all, CONGRATULATIONS for your job! The game looks GORGEOUS even on my humble Samsung S6 lite tablet :) 

I just have an issue with the cHTS file from EvilGames: everything is perfect BUT the texture in Hyrule Field: it looks "striped"... I've seen that it has been fixed in the last version but the texture is still wrong... Is that the last one?

I've tried to install the uncompressed one but unfortunally M64 PluS FZ in my tablet can't open it :(

Thank you and sorry for my english

**Comment by GhostlyDark** [comment]
The uncompressed HTS file is inside a .7z file. You'd need to decompress that archive and choose the .hts file to be imported into mupen64plus-fz. Mind you that it is currently over 22 GB in size.

Unfortunately, I have no idea what texture you are referring to. It would be nice if you could show screenshots of what exactly is going on and what exact texture you are having an issue with.

**Comment by ninobaldan** [comment]
Thank you for the quick answer!
I've tried to open the game with 240p resolution...and the texture was ok! Then I switched to 720p/1080p and it was ok again!
But opening M64plus fx with 720p set caused Hyrule field to be seen uncorrectly... very strange, maybe I have something wrong in my settings... this is the wrong texture (just to report it, sorry for the cut screenshot).
By the way, changing the resolution on-the-go can play without issues :)
![the_legend_of_zelda-001](https://user-images.githubusercontent.com/59826323/115020793-59f16f80-9ebb-11eb-8eac-fde22c9e0c39.png)
